.github/workflows/pull-request-title.yml

```
@@ -0,0 +1,13 @@
+name: pull request title check
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  pull-request-title-check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: fastify/action-pr-title@v0
+      with:
+        prefixes: 'feature:,chore:,fix:,docs:'
```

```suggestion
        prefixes: 'build:,chore:,ci:,docs:,feat:,fix:,perf:,refactor:,style:,test:'
```
Adds all the conventional commit prefixes. 😊

---

Gonna catch'em all.